### ISSUE TITLE
Incorrect version number

### ISSUE THREAD

**Issue by anfimovir** [novnc/noVNC#1039]
/docs/VERSION

1.0.0-testing.2

https://github.com/novnc/noVNC/blob/master/docs/VERSION

Current version:
1.0.0

May be simple not use additional file for prevent confusions?

**Comment by anfimovir** [comment]
Version available in:
package.json

**Comment by samhed** [comment]
Thanks

Updated /docs/VERSION in a07d4abe1f5a6ac31a73c98ee92489ed0401a5cd

**Comment by CendioOssman** [comment]
Shouldn't we remove VERSION? Is it used for anything? If we're keeping it then we need to update the release instructions.

**Comment by samhed** [comment]
Updated the release instructions.

I'd prefer keeping VERSION, even if we don't use it for anything. Feels like a common and very clear way to store the version number. However, if the rest of you want it gone I don't mind much.

While looking at `docs/` I saw that we have an old `release.txt` there that mentions VERSION. That one should probably be removed since we have release instructions in the wiki now?
